app/models/user.py

```diff
@@ -0,0 +1,24 @@
+from app import db
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
+
+class User(db.Model):
+    id = db.Column(db.BigInteger, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(250), nullable=False)
+    email = db.Column(db.String(100), index=True, unique=True, nullable=False)
```

Comment:
Could you please explain how setting index True in the email field can improve the DB performance?

Reply:
When i was saw the tutorial video, he said index=true use for login. because in login we use email and with the index=True searching the email make faster and more efficient